### PR TITLE
Hi

### PR DESCRIPTION
**Pull request by EmmaKerklingh**
Hiiiiiiiiiiii does this work?

### CODE REVIEW

**Comment by RoelandvanWijk** [comment]
Yess! Let me review :D

File: sequioa/crown.md [hunk 1]
```diff
@@ -10,3 +10,6 @@ Provide the support and connection between the higher level sub-modules and the
 
 Display green colour.
 Handle chemical data conversion and output.
+
+
+Hi hiiii
```
Comment (RoelandvanWijk) [inline sequioa/crown.md:15]: The leaves DO NOT need to be curly. Requirements management and customer reviews led to a new design, we want triangular leaves.

File: sequioa/trunk.md [hunk 2]
```diff
@@ -14,3 +14,9 @@ Greatly improve resistance to overheating in fire situation.
 ### Redcuded sap content
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+
+## Water storage
+
+Store water 101
+When: now
```
Comment (RoelandvanWijk) [inline sequioa/trunk.md:22]: Water storage needs to be in old nuclear reactor vats. We want to call it H2O Radio.

**Comment by bkarpati** [comment]
Thanks for participating in the Git workshop. I'm going to close this PR for now and also delete the remote branch.
Please stay tuned for the follow-up.

**Pull request closed without merging**